.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+	"env": {
+		"browser": true,
+		"commonjs": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+		"comma-dangle": [
+			"warn",
+			"only-multiline",
+		],
+//		"indent": [
+//			"error",
+//			"tab",
+//			{
+//				"SwitchCase": 1,
+//				"outerIIFEBody": 0,
+//			}
+//		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
+		"no-inner-declarations": "off",
+		"no-mixed-spaces-and-tabs": "error",
+		"no-redeclare": "off",
+		"no-sparse-arrays": "off",
+		"no-unused-vars": [
+			"warn",
+			{
+				"args": "none",
+				"varsIgnorePattern": "^_",
+			}
+		],
+		"no-useless-escape": "off",
+//		"quotes": [
+//			"warn",
+//			"single",
+//			{
+//				"avoidEscape": true,
+//			}
+//		],
+		"semi": [
+			"error",
+			"always",
+		],
+	},
+	"overrides": [
+		{
+			"files": [
+				"components/prism-core.js",
+				"prism.js",
+			],
+			"globals": {
+				"WorkerGlobalScope": "readonly",
+			},
+		},
+		{
+			"files": [
+				"tests/**/*.js"
+			],
+			"env": {
+				"node": true,
+				"mocha": true,
+			},
+		},
+		{
+			"files": [
+				"gulpfile.js",
+			],
+			"env": {
+				"es6": true,
+				"node": true,
+			},
+			"parserOptions": {
+				"ecmaVersion": 2015,
+			},
+			"rules": {
+				"comma-dangle": [
+					"error",
+					"always-multiline",
+				],
+			},
+		},
+		{
+			"files": [
+				"code.js",
+				"download.js",
+				"examples.js",
+			],
```

Comment:
I believe these files will need to change as a result of #1781.

Reply:
Yeah, they will…